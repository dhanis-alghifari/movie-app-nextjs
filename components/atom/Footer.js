import React from "react";

export default function Footer() {
  return (
    <>
      <footer className="relative bg-slate-200 pt-8 pb-6 mt-10">
        <div className="container mx-auto px-4">
          <div className="flex flex-wrap items-center md:justify-between justify-center">
            <div className="w-full md:w-4/12 px-4 mx-auto text-center">
              <div className="text-sm text-slate-500 font-semibold py-1">
                Copyright Â© {new Date().getFullYear()} Movie App by
                <a className="text-slate-500 hover:text-slate-800">
                  Dhanis Al Ghifari
                </a>
                .
              </div>
            </div>
          </div>
        </div>
      </footer>
    </>
  );
}
