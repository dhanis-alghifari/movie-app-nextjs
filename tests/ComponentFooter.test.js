import { render } from "@testing-library/react";
import React from "react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "../components/atom/Footer";

describe("Test if the Footer component is working as expected", () => {
    it("<Copyright /> matches snapshot", () => {
        const component = render(<Footer />);
        expect(component.container).toMatchSnapshot();
    });
});