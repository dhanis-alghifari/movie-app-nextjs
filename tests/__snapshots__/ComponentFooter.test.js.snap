// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if the Footer component is working as expected <Copyright /> matches snapshot 1`] = `
<div>
  <footer
    class="relative bg-slate-200 pt-8 pb-6 mt-10"
  >
    <div
      class="container mx-auto px-4"
    >
      <div
        class="flex flex-wrap items-center md:justify-between justify-center"
      >
        <div
          class="w-full md:w-4/12 px-4 mx-auto text-center"
        >
          <div
            class="text-sm text-slate-500 font-semibold py-1"
          >
            Copyright Â© 
            2023
             Movie App by
            <a
              class="text-slate-500 hover:text-slate-800"
            >
              Dhanis Al Ghifari
            </a>
            .
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
